examples/aws-garnet/scenario-1-deployment/aws-garnet-iot-module/test/stf-core.test.ts

```diff
@@ -0,0 +1,17 @@
+// import * as cdk from 'aws-cdk-lib';
+// import { Template } from 'aws-cdk-lib/assertions';
+// import * as Garnet from '../lib/garnet-stack';
+
+// example test. To run these tests, uncomment this file along with the
+// example resource in lib/garnet-stack.ts
+test('SQS Queue Created', () => {
```

Comment:
It looks like there are a lot of things in this aws-garnet-iot-module. Would be better to have some tests here

Reply:
I'll create an issue in the main Garnet Framework repo (https://github.com/awslabs/garnet-framework) for us to develop this. Currently, the infrastructure as code is tested by our internal team before publishing to the public repo by deploying to our accounts and validating with a set of fixed data ingestion scenarii